### ISSUE TITLE
BrowserStack - Cypress tests [13pt]

### ISSUE THREAD

**Issue by SantosGuillamot** [frontity/frontity#629]
We are going to use BrowserStack to run our tests in different OS and browsers, to ensure that Frontity is reliable in all the scenarios. ~As part of this, we are also updating Babel.~

Branch: https://github.com/frontity/frontity/tree/update-babel-polyfill

**Comment by luisherranz** [comment]
I have moved the Babel update to another PR so they are not that big.

I have also reduced the points from 21 to 13 and I will add another issue with 13 for the Babel update.

**Comment by luisherranz** [comment]
I've started working on this again.

I switched to a branch named `browserstack-cypress`: https://github.com/frontity/frontity/compare/browserstack-cypress

There have been some changes in Browserstack. Now it supports Cypress 6 and a `--sync` option in the CLI, so I will investigate that.

I think I can prepare and then explain this enough so it can be taken over by any other member of the team.

I'll try to work on it tomorrow 🙂
